client/src/pages/ProfileDetails/ProfileDetails.tsx

```diff
@@ -50,47 +63,69 @@ export default function ProfileDetails(): JSX.Element {
           flexDirection="column"
         >
           <Box borderRadius={2} className={classes.coverImage}>
-            <AvatarDisplay
-              width={windowWidth < 600 ? 100 : 150}
-              height={windowWidth < 600 ? 100 : 150}
-              loggedIn
-              user={{ name: 'example', email: 'example@example.com' }}
-            />
+            {profile ? (
+              <AvatarDisplay
+                width={window.innerWidth < 600 ? 100 : 150}
+                height={window.innerWidth < 600 ? 100 : 150}
+                loggedIn
+                user={{ name: profile.name, email: 'example@example.com' }}
+                photoUrl={profile ? profile.photo : ''}
+              />
+            ) : null}
           </Box>
           <Box m={windowWidth < 600 ? '1rem 0' : '5rem 0 1rem 0'} textAlign="center">
             <Typography fontWeight="bold" component="h1" fontSize="1.4rem">
-              {mockProfile.name}
+              {profile ? profile.name : 'No profile found'}
```

Comment:
Why don't we consider rendering the entire Profile Details card only when a profile exists? In no profile is found we can just render large text - 'No Profile Found' (rather than displaying the cover photo and profile card) 😄

Reply:
That is much better than what I did. Thanks